ceph-releases/luminous/ubuntu/16.04/daemon/start_mgr.sh

```diff
@@ -11,7 +11,7 @@ function start_mgr {
     check_admin_key
 
     # Create ceph-mgr key
-    ceph "${CLI_OPTS}" auth get-or-create mgr."$MGR_NAME" mon 'allow *' -o "$MGR_KEYRING"
+    ceph "${CLI_OPTS}" auth get-or-create mgr."$MGR_NAME" mon 'allow profile mgr' -o "$MGR_KEYRING"
```

Comment:
I think we are missing osd and mds too, see: http://docs.ceph.com/docs/master/mgr/administrator/#manual-setup